internal/cli/up.go

```diff
@@ -23,6 +24,8 @@ func (c *upCmd) register() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 	}
 
+	cmd.Flags().BoolVar(&c.watch, "watch", false, "any started services will be automatically redeployed when files in their repos change")
```

Comment:
nit: they won't necessarily be redeployed, but they will definitely be updated with their new code.

Reply:
In my mind, pushing the new code to production might not be an image or k8s deployment, but it is a service deployment. I don't know if that would be the common understanding.